### PR TITLE
Fix coping module-chart in serverless-operator

### PR DESCRIPTION
**Pull request by MichalKalke**
**Description**

Changes proposed in this pull request:

- Fix coping `module-chart` in Serverless-Operator

**Related issue(s)**
https://github.com/kyma-project/serverless-manager/issues/344

### CODE REVIEW

**Review by pPrecel — DISMISSED** [review]

**Review by halamix2 — APPROVED** [review]

**Review by halamix2 — CHANGES_REQUESTED** [review]

File: prow/jobs/modules/internal/serverless-manager.yaml [hunk 1]
```diff
@@ -599,7 +599,7 @@ presubmits: # runs on PRs
               - "sh"
             args:
               - "-c"
-              - "cp -r config/serverless/ module-chart && /image-builder --name=serverless-operator --config=/config/kaniko-build-config.yaml --context=. --dockerfile=Dockerfile"
+              - "mkdir -p module-chart && cp -r config/serverless/* module-chart && /image-builder --name=serverless-operator --config=/config/kaniko-build-config.yaml --context=. --dockerfile=Dockerfile"
```
Comment (halamix2) [inline prow/jobs/modules/internal/serverless-manager.yaml:602]:
```suggestion
              - "mkdir -p module-chart && cp -r config/serverless/\* module-chart && /image-builder --name=serverless-operator --config=/config/kaniko-build-config.yaml --context=. --dockerfile=Dockerfile"
```

File: prow/jobs/modules/internal/serverless-manager.yaml [hunk 2]
```diff
@@ -1147,7 +1147,7 @@ postsubmits: # runs on main
               - "sh"
             args:
               - "-c"
-              - "cp -r config/serverless/ module-chart && /image-builder --name=serverless-operator --config=/config/kaniko-build-config.yaml --context=. --dockerfile=Dockerfile --tag=$PULL_BASE_REF"
+              - "mkdir -p module-chart && cp -r config/serverless/* module-chart && /image-builder --name=serverless-operator --config=/config/kaniko-build-config.yaml --context=. --dockerfile=Dockerfile --tag=$PULL_BASE_REF"
```
Comment (halamix2) [inline prow/jobs/modules/internal/serverless-manager.yaml:1150]:
```suggestion
              - "mkdir -p module-chart && cp -r config/serverless/\* module-chart && /image-builder --name=serverless-operator --config=/config/kaniko-build-config.yaml --context=. --dockerfile=Dockerfile --tag=$PULL_BASE_REF"
```

File: prow/jobs/modules/internal/serverless-manager.yaml [hunk 3]
```diff
@@ -1287,7 +1287,7 @@ postsubmits: # runs on main
               - "sh"
             args:
               - "-c"
-              - "cp -r config/serverless/ module-chart && /image-builder --name=serverless-operator --config=/config/kaniko-build-config.yaml --context=. --dockerfile=Dockerfile --tag=$PULL_BASE_SHA"
+              - "mkdir -p module-chart && cp -r config/serverless/* module-chart && /image-builder --name=serverless-operator --config=/config/kaniko-build-config.yaml --context=. --dockerfile=Dockerfile --tag=$PULL_BASE_SHA"
```
Comment (halamix2) [inline prow/jobs/modules/internal/serverless-manager.yaml:1290]:
```suggestion
              - "mkdir -p module-chart && cp -r config/serverless/\* module-chart && /image-builder --name=serverless-operator --config=/config/kaniko-build-config.yaml --context=. --dockerfile=Dockerfile --tag=$PULL_BASE_SHA"
```

**Review by halamix2 — CHANGES_REQUESTED** [review]

File: prow/jobs/modules/internal/serverless-manager.yaml (hunk 1 above)
Comment (anoipm) [reply prow/jobs/modules/internal/serverless-manager.yaml:602]: Why? This is shell command - not regexp.

**Pull request merged**